legend-engine-language-pure-dsl-service/src/test/java/org/finos/legend/engine/language/pure/grammar/test/TestServiceGrammarParser.java

```diff
@@ -1124,4 +1124,65 @@ public void testServiceTestSuitesErrorMessages()
                 "PARSER error at [43:17-22]: Unexpected token 'actual'"
         );
     }
+
+    @Test
+    public void testInlineServiceExecution()
```

Comment:
Can we break these into 2 separate tests ? The first one seems to assert that we throw a parse error because the service does not inline the mapping and runtime even though the execution type is Inline ; vs the second test is the happy path parsing test